pkg/cloud/helpers.go

```diff
@@ -37,6 +38,12 @@ func setArrayIfNotEmpty(strArray []string, setFn setArray) {
 	}
 }
 
+func setIntIfPositive(num int64, setFn setInt) {
```

Comment:
Look at us go!

I bet generics will make this sort of things much easier someday.